Clarification for the MergeUniqueNode operator (#1489)
neo4j/docs-cypher: modules/ROOT/pages/planning-and-tuning/operators/operators-detail.adoc

----
////

[NOTE]
====

> `MergeUniqueNode` has been added in Cypher 2025.11 but it did not work as intended in that version.

Reviewer: ```suggestion
`MergeUniqueNode` was added in Cypher 2025.11 but it did not work as intended in that version.
```

> For this reason, the operator has been disabled in Cypher 2025.12.

Reviewer: ```suggestion
For this reason, the operator was disabled in Cypher 2025.12.
```

> Cypher 2026.02 contains a fix which makes it the first Cypher version with the operator working as intended. 

Reviewer: ```suggestion
Cypher 2026.02 fixes the bug and is thus the recommended version to use. 
```
Or something like that.
